src/Redux/books/books.js

```diff
@@ -0,0 +1,24 @@
+export const initialState = [];
+
+export const addBook = (payload) => ({
+  type: 'ADD_BOOK',
+  payload,
+});
+export const removeBook = (payload) => ({
+  type: 'REMOVE_BOOK',
+  payload,
+});
+
+const reducer = (state = initialState, action) => {
+  switch (action.type) {
+    case 'ADD_BOOK':
+      return [...state, action.payload];
+    case 'REMOVE_BOOK':
```

Comment:
- [ ] As stated in the requirements, your actions should be defined using the **ducks pattern**. Kindly defined your `Action Types` to be constants (see image) and then use those constants in your `Action Creators` and `reducer`.

  ![muyo-ducks](https://user-images.githubusercontent.com/36542986/150335168-b31bfe0a-0d72-4e53-8896-acf65275b5a0.jpg)